src/OpenTelemetry.Contrib.Instrumentation.AWS/Implementation/AWSMessagingUtils.cs

```diff
@@ -0,0 +1,35 @@
+// <copyright file="AWSMessagingUtils.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+using OpenTelemetry.Context.Propagation;
+
+namespace OpenTelemetry.Contrib.Instrumentation.AWS.Implementation;
+
+internal static class AWSMessagingUtils
```

Comment:
A separate class is probably overkill now, this Inject method could be defined inside AWServiceType now (BTW, I'm also not sure if the split of ServiceType from ServiceHelper is helpful)

Reply:
Lets keep code more structured. Probably we will have to support even more AWS triggers with it's types. Otherwise the single method of this class will go into AWSTracingPipelineHandler as well as possible future methods for other AWS triggers. AWSTracingPipelineHandler will become big all-in-one class.